ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/TaskHistoryMapper.java

```diff
@@ -95,6 +95,8 @@ List<TableTaskHistory> selectTaskHistory(@Param("tableIdentifier") TableIdentifi
       @Result(column = "table_name", property = "tableIdentifier.tableName"),
       @Result(column = "task_group_id", property = "taskGroupId"),
       @Result(column = "task_history_id", property = "taskHistoryId"),
+      @Result(column = "task_trace_id", property = "taskTraceId"),
+      @Result(column = "task_attempt_id", property = "taskAttemptId"),
```

Comment:
`selectTaskHistoryByQueueIdAndTime` is never used, may be useless.

Reply:
ok

Reply:
> ok, i will remove useless code